code/game/objects/items/devices/broadcast_camera.dm

```diff
@@ -49,6 +41,17 @@
 	icon_state = "[base_icon_state]0"
 	return ..()
 
+/obj/item/broadcast_camera/attack_self(mob/user, modifiers)
+	. = ..()
+	activate = !activate
+	icon_state = "[base_icon_state][activate]"
+	if(activate)
+		on_activating()
+		playsound(src, 'sound/misc/radio_talk.ogg', 100, TRUE)
+	else
+		on_deactivating()
+		playsound(src, 'sound/misc/radio_receive.ogg', 100, TRUE)
```

Comment:
Does it really need any additional sounds? It already plays sound in the `on_activating()` and `on_deactivating()` procs which get called.

Reply:
Removed. Didn't hear it becouse volume 20